src/pages/conference/timeline/_schedule.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react'
+import styled from 'styled-components'
+import ArrowRight from './images/downwards_right.svg'
+import ArrowLeft from './images/downwards_left.svg'
+import { Header } from 'components/elements'
+import device from 'themes/device'
+
+type ScheduleProps = {
+    time?: string | number
+    id?: string
+    name?: string
+    topic?: string
+}
+
+type DataProps = {
+    timings: ScheduleProps[]
+    free_time: string | number
+    speakers: ScheduleProps[]
+}
+
+type ScheduleConferenceProps = {
+    id?: string
+    title?: string
+    date?: string
+    period?: string
+    data?: DataProps[]
+}
+
+type ConferenceProps = {
+    item: ScheduleConferenceProps
+    key: string
+}
+
+const MainContainer = styled.div`
+    display: flex;
+    justify-content: center;
+`
+
+const Container = styled.div`
+    padding-bottom: 80px;
+`
+
+const HeaderTitle = styled.div`
+    padding-bottom: 80px;
+`
+
+const WrapperContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    max-width: 995px;
+`
+
+const ImageContainer = styled.div`
+    @media ${device.tablet} {
+        margin-left: 40px;
+    }
+`
+
+const TimeWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    h2:last-child {
```

Comment:
Is this behavior works after change `<Header as="h2 />` by default on `<Header as="div" />` in current implementation. If not, let's remove it

Reply:
we need to change it back to h2 only